### DISCUSSION TITLE
posts/scammed-by-first-freelance-job

### DISCUSSION THREAD

**Discussion by giscus** [sahilrajput03/sahilrajput.ml#6] [Announcements]
# posts/scammed-by-first-freelance-job

I like to test 🤺︎, explore 🥝︎ and create 🥇︎ better softwares.

https://sahilrajput.ml/posts/scammed-by-first-freelance-job

**Comment by Tekipeps** [comment]
I'm glad you were able to get to the root of this. Nicely done ^^

**Reply by sahilrajput03** [reply]
Thanks for the feedback.

**Comment by Melford-D** [comment]
Hmmm... Wow
I like the end of the story.

**Reply by sahilrajput03** [reply]
Thanks for the feedback, Melford. :D

**Comment by negivivek1101** [comment]
Nicely executed, best you could do, great!! Scammer should be behind bars!

**Reply by sahilrajput03** [reply]
yes, writing to us embassy as well. We got his real photo as well.

 🔥

**Reply by negivivek1101** [reply]
Right

**Comment by CoreyOdonis** [comment]
This is good so no one falls victim. I pray he ends up in jail where he belongs

**Reply by sahilrajput03** [reply]
Yes, he will end up very soon. :) Thanks for response.
